src/app/templates/activity/activity-item/activity-type-view.tsx

```diff
@@ -39,8 +39,8 @@ export const ActivityTypeView: FC<Props> = ({ activity }) => {
   } else {
     contents = (
       <>
-        <T id={activityTypesI18nKeys[activity.type]} />
-        {isInteraction && (
+        <T id={is3Route ? 'route3' : activityTypesI18nKeys[activity.type]} />
```

Comment:
I think you should create a dictionary of translations for most of interaction subtypes and use it here instead of `is3RouteFlag`; it will make adding Quipuswap, Plenty, etc. easier